lib/statsd/instrument/environment.rb

```diff
@@ -22,6 +22,15 @@ def default_backend
     end
   end
 
+  def datagram_builder_class
+    case ENV['STATSD_IMPLEMENTATION']
+    when 'datadog', 'dogstatsd'
+      StatsD::Instrument::DogStatsDDatagramBuilder
+    else
```

Comment:
Shouldn't we also have `case 'statsite'`?  Since we still have the statsite specific `key_value` method, I assume we are still supporting it.

Reply:
it should, haven't got around to implement it yet.